editor/src/clj/editor/validation.clj

```diff
@@ -93,6 +93,10 @@
     (when (not (<= min v max))
       (util/format* tmpl name min max))))
 
+(defn prop-collision-shape-conflict? [shapes collision-shape]
+  (when (and collision-shape (not (empty? shapes)))
+    "Cannot use both 'Collision Shape' and 'Shapes'. Please remove one of them."))
+
```

Comment:
Since 'Shapes' isn't a propery the user can see anywhere, but rather the list of shape objects they've added in the Outline view, maybe the error message could be clearer?

What do you think about `"Cannot combine embedded shapes with a referenced 'Collision Shape'. Please remove either."`?

Reply:
I agree, your suggestion sounds much better. Thank you